Sources/CryptorRSA/CryptorRSA.swift

```diff
@@ -309,6 +309,18 @@ public class CryptorRSA {
             let rsaEncryptCtx = EVP_CIPHER_CTX_new_wrapper()
             EVP_CIPHER_CTX_init_wrapper(rsaEncryptCtx)
             
+			// get rsaKey
+			guard let rsaKey = EVP_PKEY_get1_RSA(.make(optional: key.reference)) else {
+				let source = "Couldn't create key reference from key data"
+				if let reason = CryptorRSA.getLastError(source: source) {
+					throw Error(code: ERR_ADD_KEY, reason: reason)
+				}
+				throw Error(code: ERR_ADD_KEY, reason: source + ": No OpenSSL error reported.")
+			}
+			defer {
+				RSA_free(rsaKey)
+			}
+			
```

Comment:
This additional block exists now because we are storing the envelope key rather than the RSA key.

Changing the key to store the envelope key rather than the RSA key it contains is beneficial because the sign, verify and CBC decrypt functions no longer need to construct a new one (and copy the RSA key in), which should shorten their pathlength.  (caveat: we did not benchmark this)